arviz/rcparams.py

```diff
@@ -93,23 +102,37 @@ def _validate_boolean(value):
     return value is True or value == "true"
 
 
-def _validate_tuple(value):
-    """Validate value is a tuple of size 2."""
-    if value is None:
-        return value
+def _validate_float_iterable(value, length):
+    """Validate value is a float iterable of given length."""
+    if value is None or isinstance(value, str) and value.lower() == "none":
+        return None
     elif value == "auto":
         return value
-    elif isinstance(value, tuple) and len(value) == 2:
-        return value
+    elif isinstance(value, str):
+        try:
+            val = [_validate_float_or_none(v.strip("( )")) for v in value.split(',') if v.strip()]
+            if len(val) == length:
+                return tuple(val)
+            else:
+                raise ValueError("Iterable must be of length: {}".format(length))
+        except Exception:
+            raise
+    elif np.iterable(value) and not isinstance(value, (set, frozenset)):
+        val = [_validate_float_or_none(v) for v in value]
+        if len(val) == length:
+            return tuple(val)
+        else:
+            raise ValueError("Iterable must be of length: {}".format(length))
```

Comment:
As rcParams are stll quite new, I would try to divide the code in as many potentially useful building blocks as possible. For example, a function similar to matplotlib's `listify_validator` will probably be used in the future, so it would be interesting to have it in an independent function that can then be called from `validate_float_iterable` and any other function to come.

Reply:
Got it, I'll make a more general ```_validate_iterable``` function to validate any iterable data types and then just call it in ```_validate_float_iterable``` in this case. Am I right?